src/radical/pilot/pilot_manager.py

```diff
@@ -608,10 +607,6 @@ def submit_pilots(self, descriptions):
             with self._pilots_lock:
                 self._pilots[pilot.uid] = pilot
 
-            if self._session._rec:
-                ru.write_json(pd.as_dict(), "%s/%s.batch.%03d.json"
-                        % (self._session._rec, pilot.uid, self._rec_id))
```

Comment:
there is a leftover of `self._rec_id` in `__init__` (in both PilotManager and TaskManager)

Reply:
Thanks - removed.